exercises/practice/equilateral_triangle/equilateral_triangle.py

```diff
@@ -0,0 +1,21 @@
+from z3 import *
+
+def generate_third_points(first_point, second_point):
+    pass
+
+def convert_to_float(z3_number):
```

Comment:
Maybe add a comment above this function indicating to the user that they shouldn't change it.